src/main/kotlin/com/terraformation/backend/i18n/TimeZones.kt

```diff
@@ -0,0 +1,503 @@
+package com.terraformation.backend.i18n
+
+import com.squarespace.cldrengine.CLDR
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.TextStyle
+import java.util.*
+import java.util.concurrent.ConcurrentHashMap
+import org.apache.commons.text.similarity.LevenshteinDistance
+
+object TimeZones {
+
+  private val chromeTimeZones =
+      listOf(
```

Comment:
should this be a separate data file of sorts?

Reply:
I did end up moving this to a JSON file, yeah. Maybe at some point we'll set up a fancy CI pipeline that auto-updates it as new browser versions are released.